commands/slash/247.js

```diff
@@ -2,6 +2,9 @@ const { MessageEmbed } = require("discord.js");
 const SlashCommand = require("../../lib/SlashCommand");
 
 const command = new SlashCommand()
+   let channel = await client.getChannel(client, interaction);
+   if (!channel) return;
```

Comment:
This will slolve the simple problem of checking if the user is in the channel or no. But It would be better if you instead use https://github.com/SudhanPlayz/Discord-MusicBot/blob/v5/commands/slash/loop.js#L8-L35 that type of checking so the user/member can know whats wrong with it.